### PR TITLE
grind script cleanup

### PR DESCRIPTION
**Pull request by devoncarew**
- misc changes to the grind script - mostly some upgrades from grinder 0.7.0

### CODE REVIEW

**Comment by keertip** [comment]
lgtm! Thanks!

File: tool/grind.dart [hunk 1]
```diff
@@ -167,6 +163,6 @@ Future _uploadStats(GrinderContext context, int sdkDocsGenTime) async {
         description: 'Commit ${commit}', links: [link]);
     return librato.createAnnotation('build_ui', annotation);
   } else {
-    return true;
+    return new Future.value();
```
Comment (sethladd) [inline tool/grind.dart]: if this is an async function, why not just return ?

Comment (devoncarew) [inline tool/grind.dart]: sgtm

**Comment by devoncarew** [comment]
updated -

**Pull request merged**